packages/analytics/analytics-chart/src/styles/base.scss

```diff
@@ -1 +1,2 @@
-@import "./mixins";
+@forward "@kong/design-tokens/tokens/scss/variables";
```

Comment:
issue: these should be `@use` not `@forward`: https://github.com/sass/sass/issues/3782#issuecomment-2421142866

Reply:
I saw that comment the other day and I wasn't quite following why we shouldn't use `@forward`. According to the [docs](https://sass-lang.com/documentation/at-rules/forward/), `@forward "foo"` should work similar to `export * from 'foo'` in JavaScript modules.

In this case changing the `@forward` here to `@use` does not work. `base.scss` acts like a barrel file in JS here.

Reply:
forward should be for mixins, functions, whereas use should be variables and "includes" as I understand it

Reply:
But you can't import variables into the global namespace with `@forward`? There is no namespace

Reply:
```scss
// vars.scss
$heading-color: red;

// mixins.scss
@mixin heading {
  font-weight: 500;
  font-size: 20px;
}

// lib.scss
@forward "vars";
@forward "mixins";

// app.scss
@use "lib" as *;

h3 {
  @include heading;
  color: $heading-color;
}
```

Reply:
There is no more global namespace with `@use` so the best we can do is to inject variables to style file with Vite (those `@use`ed or `@forward`ed files won't be processed by Vite).

Reply:
> namespace

well there is, with `as *` 😬

Reply:
I mean there's no global namespace that can work across files. Anyway, I think it is correct to use `@forward` to compose a "library file" here.

And I don't quite like the file name either as it kind of implies that there is style output in it, which isn't the case. But as it named like this before, I'd say lets keep it as-is here.

Reply:
if `base` is just mixins/variables, feel free to rename them in this PR to clear things up as long as functionality remains the same

Reply:
Do we have a naming convention? Like "lib.scss"?

Reply:
No, we try to name more explicitly, e.g. `mixins` or `variables` etc.

In this case it could likely just be something like `globals` since that's what it is?

Reply:
👍

Reply:
Renamed `base.scss` to `globals.scss`.